mygnupg.py

```diff
@@ -0,0 +1,64 @@
+import gnupg
+
+class mygnupg():
+    """
+    This class used for initial gnupg, 
+    create, import, and export key pairs.
+    Also, can encrypt and decrypt data.
+    """
+    def __init__(self):
+        """
+        The binary file should be different
+        in different os.
+        This is MacOS version.
+        """
+        self.gpg = gnupg.GPG(gnupghome='./gnupgkeys')
+        self.key = None
+        self.email = "john_smith@scmail.org"
+        self.keytype = "RSA"
+        self.password = "sample"
+
+    def import_key(self, path):
+        """
+        This function used for check whether the gnupg pub or pvt key already exist.
+        If exist, import it, if not, return false.
```

Comment:
This method currently raises an error if the key does not exist.

Reply:
Yeah, I want the program does that, or should we use other methods to notify the user there is no key pair import?